pkgs/applications/misc/rkvm/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, rustPlatform, fetchFromGitHub
+, pkg-config, libevdev, openssl, llvmPackages_latest, linuxHeaders }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rkvm-unstable";
+  version = "ec404c69";
+
+  src = fetchFromGitHub {
+    owner = "htrefil";
+    repo = pname;
+    rev = "ec404c69b38f7feff5103f898612734ee8d7ee95";
+    sha256 = "sha256-YZXHbZ71EsSyAtTenQ4rgp1fJbatkk7BW4Cmr/RpyXc=";
+  };
+
+  cargoSha256 = "sha256-1sxP74xt3DGCAhY6Dolaz2JB3Q6CTvtkgln08AMGmc0=";
+
+  postPatch = ''
+    sed -i 's|.clang_arg("-I/usr/include/libevdev-1.0/")|.clang_arg("-I${libevdev}/include/libevdev-1.0").clang_arg("-I${linuxHeaders}/include")|g' ./input/build.rs
```

Comment:
Maybe we should do a patch for this because this seems quite complex.

Reply:
Hi @SuperSandro2000, thanks for looking so fast. I've left this as a draft as someones helping me in IRC. I plan to file bugs upstream, wait for them, something before marking this as ready. Or at least writing this in a slightly more readable way at least.